queries.md

```diff
@@ -4,76 +4,96 @@
 
 ### 1. All the companies whose name match 'Babelgum'. Retrieve only their `name` field.
 
-<!-- Your Code Goes Here -->
+filter: {name: 'Babelgum'}
+project: {name:1}
 
 ### 2. All the companies that have more than 5000 employees. Limit the search to 20 companies and sort them by **number of employees**.
 
-<!-- Your Code Goes Here -->
+filter: {number_of_employees: {\$gt: 5000} }
+project:
+sort: {number_of_employees:1}
+Collaboration:
+Limit: 20
 
 ### 3. All the companies founded between 2000 and 2005, both years included. Retrieve only the `name` and `founded_year` fields.
 
-<!-- Your Code Goes Here -->
+filter: {$and: [{founded_year:{$gte:2000}}, {founded_year:{\$lte:2005}}]}
```

Comment:
You can save yourself here some keystrokes:
```js
founded_year: {  $gte: 2000, $lte: 2005  }
```

Reply:
Brill!